hack/olm.sh

```diff
@@ -42,7 +42,6 @@ source $WMCO_ROOT/hack/common.sh
 
 cd $WMCO_ROOT
 OSDK=$(get_operator_sdk)
-VERSION=$(get_version)
```

Comment:
Why is this change needed? A question like this implies that it warrants a mention in the commit message.

Reply:
changed the commit message